demo-invenioils/ui/src/overridden/frontsite/Home/Sections/SectionDocsAndUsage.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Card, Container, Header, Label, List, Grid } from 'semantic-ui-react';
+
+export const SectionDocsAndUsage = () => {
+  return (
+    <Container fluid className="dot-background-container">
+      <Container fluid className="dot-background">
+        <Container className="fs-landing-page-section">
+          <Header
+            as="h1"
+            className="section-header highlight"
+            textAlign="center"
+          >
+            Documentation and usage
+          </Header>
+          <Grid centered stackable>
+            <Grid.Row>
+              <Grid.Column width={12}>
```

Comment:
What is the purpose of the width 12 when you have only one column? I'm not even sure if you need to use Grid.Column when you just have one column.

Reply:
it is to nicely limit the width of the child components without adding custom css, it was agreed with me priorly